org.adishatz.syncpasswd.json

```diff
@@ -0,0 +1,129 @@
+{
+    "app-id" : "org.adishatz.syncpasswd",
```

Comment:
is this domain controlled by the upstream developers?

Reply:
Yes, this is my domain, see https://gitlab.gnome.org/GNOME/geary/-/blob/main/src/client/conversation-list/conversation-list-view.vala?ref_type=heads#L3